client/src/components/Nav.tsx

```
@@ -105,15 +103,12 @@ function Nav() {
               <img src={dashboardIcon} alt="Dashboard" className="w-8 px-1" />
               <Link to="/dashboard">Dashboard</Link>
             </li>
-            <li
-              className="flex flex-row px-2 py-1 m-1 rounded-md border-2 hover:font-bold sm:m-0 grow hover:bg-pink-400"
-            >
+            <li className="flex flex-row px-2 py-1 m-1 rounded-md border-2 hover:font-bold sm:m-0 grow hover:bg-pink-400">
               <img src={logoutIcon} alt="Logout" className="w-8 px-1" />
               <button onClick={handleLogout}>Logout</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)